### ISSUE TITLE
Is openblocks closing down?

### ISSUE THREAD

**Issue by slahiri** [openblocks-dev/openblocks#376]
Hi,
We were evaluating openblocks seriously for our project when we started getting messages that online version will be brought down. Will you guys be continuing with this project? I feel you have a wonderful piece of framework.

SID

**Comment by esbolice** [comment]
Hi, I just separated and created a private repo and work on it alone)

**Comment by FalkWolsky** [comment]
Hello SID, 

unfortunately the initial Developer Team got silent some weeks ago and seemed to have left the project.
However, Openblocks is such a great project, that out of the Community we do not want it to die.

Therefore we start all over / further and have now [https://lowcoder.cloud](https://lowcoder.cloud) as new home for the community version for free, as also a new repository: [https://github.com/Lowcoder-Community/openblocks](https://github.com/Lowcoder-Community/openblocks) and a new Discord Server: [https://discord.gg/usCgMKFN](https://discord.gg/usCgMKFN)

We believe in the power of the community and hope to ensure a good future for Openblocks.

**Comment by slahiri** [comment]
I agree @FalkWolsky. As part of my organization I am evaluating OpenBlocks and would like to contribute back as well. Best feature is ready integration as ReactJS without iframe - as not other module provides.

**Comment by j2l** [comment]
Great!
The Discord invite has expired @FalkWolsky

**Comment by rubenfiszel** [comment]
Windmill (https://github.com/windmill-labs/windmill) is another open-source alternative to Retool with as a consequence a lot of overlap with Openblocks. We have a lot of respect for openblock's team work. However, If openblocks were to shutdown, we would be happy to provide migration guides. We are more active than ever and with financial stability.

**Comment by abulka** [comment]
Windmill looks impressive - thanks!
